src/State/State.spec.js

```diff
@@ -137,7 +137,7 @@ test('State evalWith', t => {
   t.end()
 })
 
-test('State get', t => {
+test('State get errors', t => {
   const get = bindFunc(State.get)
 
   const err = /State.get: Function Required/
```

Comment:
Need to change the wording on this error to include language about no arguments.